packages/gamut-labs/src/LayoutMenu/LayoutMenu.tsx

```diff
@@ -0,0 +1,100 @@
+import { Box, Logo, StrokeButton } from '@codecademy/gamut';
+import React, { useState } from 'react';
+
+import { Flyout } from '../Flyout';
+import { AccordionMenu, Section, SectionItem } from './AccordionMenu';
+
+export type LayoutMenuProps = {
+  /**
+   * An array of sections including the title, slug, and items, each of which will become an accordion
+   */
+  sections: Section[];
+  /**
+   * The slug of the current selected item
+   */
+  selectedItem?: string;
+  /**
+   * Callback to be run on click of the accordion button
+   */
+  onSectionToggle: (sectionSlug: string) => void;
+  /**
+   * Callback to be run on click of a section item. This should include where the link should go
+   */
+  onSectionItemClick: (item: SectionItem, sectionSlug: string) => void;
+  /**
+   * Text shown in mobile button that opens flyout on click
+   */
+  mobileButtonText: string;
+  /**
+   * Whether the mobile button should be full width
+   */
+  fullWidthButton?: boolean;
+  /**
+   * Whether or not to show the Codecademy logo at the top of the flyout on mobile
+   */
+  showLogoOnFlyout?: boolean;
+};
+
+export const LayoutMenu: React.FC<LayoutMenuProps> = ({
+  sections,
+  onSectionToggle,
+  onSectionItemClick,
+  selectedItem,
+  mobileButtonText,
+  fullWidthButton = true,
+  showLogoOnFlyout,
+  children,
+}) => {
+  const [flyoutOpen, toggleFlyout] = useState<boolean>(false);
```

Comment:
Nit: TypeScript can infer this generic type from the `false`

```suggestion
  const [flyoutOpen, toggleFlyout] = useState(false);
```